app/konnect/getting-started/publish-service.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Set Up and Access Dev Portal
+no_version: true
+---
+
+The Dev Portal is an API catalog that lets you document your Konnect Services
+and share them with your developers. Developers can use the Dev Portal to
+locate, access, consume, and register applications against the Services.
+
+This guide walks you through setting up a sample API spec and description for
+the {{site.konnect_short_name}} service and publishing the service to the Dev
+Portal. You'll also have a chance to check out the Dev Portal live, and test
+out some customization options.
+
+## Upload a description
+
+{% include_cached /md/konnect/dev-portal.md section='markdown' %}
+
+## Upload an API spec
+
+{% include_cached /md/konnect/dev-portal.md section='spec' %}
+
+## Publish a Service
+
+1. Return to the overview page for the `example_service`.
+
+1. In the top right corner, click on the **Actions** dropdown and select
+**Publish to Portal**.
+
+    By default, this publishes all of the Service's version specs to a private
+    Dev Portal site.
+
+
+## View the published content on Dev Portal
+
+{% include_cached /md/konnect/dev-portal.md section='access-portal' %}
+
+## Customize your Dev Portal
```

Comment:
Sort of. The [appearance customization doc](https://docs.konghq.com/konnect/dev-portal/customization/appearance/) has minimal info, so linking off to it would honestly be more annoying than anything. 

Let me expand this section to give the user some options instead. I'll link off to custom domain docs to provide another customization option, at least.